src/main/java/org/kilocraft/essentials/api/command/ArgumentSuggestions.java

```diff
@@ -61,7 +61,7 @@
 
 
     public static CompletableFuture<Suggestions> allNonOperators(CommandContext<ServerCommandSource> context, SuggestionsBuilder builder) {
-        return CommandSource.suggestMatching(playerManager.getPlayerList().stream().filter((p) -> !playerManager.isOperator(p.getGameProfile())).map((p) -> p.getGameProfile().getName()), builder);
+        return CommandSource.suggestMatching(playerManager.getPlayerList().stream().filter((p) -> !playerManager.isOperator(p.getGameProfile())).map((p) -> p.getName().asString()), builder);
```

Comment:
Couldn't you use `PlayerEntity.getEntityName()` instead as that does the exact same thing with less code